crossdock/client/oneway/behavior.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package oneway
+
+import (
+	"go.uber.org/yarpc/crossdock/client/params"
+
+	"github.com/crossdock/crossdock-go"
+)
+
+// createT tags the given T with the transport, encoding and server.
+func createT(t crossdock.T) crossdock.T {
+	t.Tag("encoding", t.Param(params.Encoding))
+	t.Tag("server", t.Param(params.Server))
+	return t
```

Comment:
You don't need to follow this pattern anymore. This was a leftover from how we
were doing crossdock tests before. You can tag inside the behavior instead and
get rid of this function and file.